### DISCUSSION TITLE
Add support for deno debugger

### DISCUSSION THREAD

**Discussion by dancrumb** [supabase/supabase#13136] [Feature Requests] [answered]
It would be really useful to be able to start the deno debugger in the deno-relay container with the appropriate port exposed so that a compliant inspector can attach to it.

I'd be happy to have a go at implementing this if you agree it's worthwhile.

It would involve the following:
 - Add an argument to `supabase functions server` to enable the debugger
 - Update the code to expose the relevant port
 - Some investigation and implementation to make sure that things like breakpoints work sensibly.

**Comment by flemx** [comment]
I agree that that this would be very useful, the terminal is not sufficient for debugging and love to use the Chrome Devtools. 

In the source code I can find here on line 189 the deno parameters added: 
https://github.com/supabase/cli/blob/main/internal/functions/serve/serve.go

**Comment by laktek** [comment, marked as answer]
You can now use Chrome DevTools to debug Edge Functions https://supabase.com/docs/guides/functions/debugging-tools (also check the video tutorial on how to use it https://www.youtube.com/watch?v=sOrtcoKg5zQ)
